packages/components/src/font-size-picker/index.js

```diff
@@ -51,26 +51,34 @@ function FontSizePicker( {
 							isLarge
 							onClick={ onToggle }
 							aria-expanded={ isOpen }
-							aria-label={ __( 'Custom font size' ) }
+							aria-label={ sprintf(
+								/* translators: %s: font size name */
+								__( 'Font size: %s' ), currentFontSizeName
+							) }
 						>
-							{ ( currentFont && currentFont.name ) || ( ! value && _x( 'Normal', 'font size name' ) ) || _x( 'Custom', 'font size name' ) }
+							{ currentFontSizeName }
 						</Button>
 					) }
 					renderContent={ () => (
 						<NavigableMenu>
-							{ map( fontSizes, ( { name, size, slug } ) => (
-								<Button
-									key={ slug }
-									onClick={ () => onChange( slug === 'normal' ? undefined : size ) }
-									className={ 'is-font-' + slug }
-									role="menuitem"
-								>
-									{ ( value === size || ( ! value && slug === 'normal' ) ) &&	<Dashicon icon="saved" /> }
-									<span className="components-font-size-picker__dropdown-text-size" style={ { fontSize: size } }>
-										{ name }
-									</span>
-								</Button>
-							) ) }
+							{ map( fontSizes, ( { name, size, slug } ) => {
+								const isSelected = ( value === size || ( ! value && slug === 'normal' ) );
+
+								return (
+									<Button
+										key={ slug }
+										onClick={ () => onChange( slug === 'normal' ? undefined : size ) }
+										className={ 'is-font-' + slug }
```

Comment:
Just a little thing, but we can use template strings here:
```suggestion
										className={ `is-font-${ slug }` }
```